app/pods/publications/overview/_base/route.js

```diff
@@ -70,11 +96,8 @@ export default class PublicationsOverviewBaseRoute extends Route {
 
   setupController(controller) {
     controller.columnsDisplayConfigStorageKey = this.columnsDisplayConfigStorageKey;
-    let columnsDisplayConfig = this.loadColumnsDisplayConfig();
-    if (!columnsDisplayConfig) {
-      columnsDisplayConfig = controller.getDefaultColumnsDisplayConfig();
-    }
-    controller.columnsDisplayConfig = columnsDisplayConfig;
+    controller.columnsDisplayConfig = this.columnsDisplayConfig;
+    controller.getDefaultColumnsDisplayConfig = this.getDefaultColumnsDisplayConfig;
```

Comment:
Methods beginnen doorgeven tussen route/controller vind ik wel wat vreemd (maar an sich niet incorrect). Kunnen we misschien een utils file (of misschien zelfs class?) maken voor die colum-display-config die methods voorziet om te laden/op te slaan uit/in local storage, te resetten, default config te geven, ... Dan staat het allemaal samen ipv verspreid over  route/controller  nu.